src/languages/en.js

```diff
@@ -476,12 +479,19 @@ export default {
     setPasswordPage: {
         enterPassword: 'Enter a password',
         setPassword: 'Set password',
+        confirmPassword: 'Confirm Password',
+        confirmPasswordInvalidMessage: 'Please make sure the confirm password is match',
         newPasswordPrompt: 'Your password must have at least 8 characters, 1 capital letter, 1 lowercase letter, and 1 number.',
-        passwordFormTitle: 'Welcome back to the New Expensify! Please set your password.',
+        passwordFormTitle: 'Please set a password below to sign into your account',
```

Comment:
Please add a period `.` as per the requirement https://github.com/Expensify/App/issues/7288#issue-1106782416